mrvi/_model.py

```diff
@@ -33,13 +32,38 @@
     plan_kwargs=dict(
         lr=1e-2,
         n_epochs_kl_warmup=20,
-        do_comp=False,
-        lambd=0.1,
     ),
 )
 
 
 class MrVI(UnsupervisedTrainingMixin, VAEMixin, BaseModelClass):
+    """
+    Multi-resolution Variational Inference (MrVI) :cite:`Boyeau2022mrvi`.
```

Comment:
```suggestion
    Multi-resolution Variational Inference (MrVI) :cite:`Boyeau2022mrvi`.
```

You would need to add the sphinxcontrib bib extension. But let's not focus on the docs for this codebase